### DISCUSSION TITLE
Linear Regession Problem

### DISCUSSION THREAD

**Discussion by draconcio12** [anyoptimization/pymoo#302] [Q&A]
How to define a problem to get the parameter for a linear regression?

**Comment by blankjul** [comment]
If you are still interested, can you provide some more information?

**Comment by draconcio12** [comment]
I want to Fit a linear regression, how I define the problem to get It?

**Comment by blankjul** [comment]
Have you tried defining a least squared error problem? You basically want to minimize the squared distances to a line.
(there are more efficient ways doing that than using pymoo as the focus here is black-box optimization)
